suite-common/wallet-utils/src/accountUtils.ts

```diff
@@ -837,3 +838,33 @@ export const readUtxoOutpoint = (outpoint: string) => {
     const vout = buffer.readUInt32LE(txid.length);
     return { txid: txid.toString('hex'), vout };
 };
+
+export const getExcludedUtxos = (
```

Comment:
i think it should be placed in sendFormUtils

Reply:
Moved in b9c0a769947a03f3e4b77c791951a6913240cb52